tests_e2e/tests/scripts/get_distro.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env pypy3
```

Comment:
This script is sharing across multiple tests and places, may be lib is a good place. I think only test scripts are placing it here

Reply:
lib is a python module; 'scripts' is the proper place for bash scripts